Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM ruby:2.4.4-alpine3.7
```

Comment:
Can this move to aws/Dockerfile, or will that cause trouble with the Docker Hub integration you were trying to set up?

Reply:
Unfortunately seems like Docker Hub can't handle this. (if running manually the build would have to be called smth like `docker build -t xyz ../`, basically the context have to be the root directory, as we need the whole repo inside)

Reply:
Ok.